jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -150,30 +153,34 @@ public class OdeProperties implements EnvironmentAware {
    private String kafkaTopicFilteredOdeSpatJson = "topic.FilteredOdeSpatJson";
    private String kafkaTopicOdeRawEncodedSPATJson = "topic.OdeRawEncodedSPATJson";
    private int spatReceiverPort = 44910;
-   private int spatBufferSize = 1000;
+   private int spatBufferSize = 500;
```

Comment:
The intent behind this change?

Reply:
SPaT is one of the smallest message types and this has generally been overkill.